dace/libraries/standard/nodes/gearbox.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
+"""
```

Comment:
Documentation should move to the node itself. This goes into vscode too so it's nice to have it there

Reply:
Ok! I don't use VSCode so I don't see it. I'll put it in the node

Reply:
Resolved in e6d1044d